app/assets/stylesheets/components/_search_filter.scss

```diff
@@ -1,19 +1,36 @@
 .search-filter {
-  border-radius: $search-filter-radius;
+  border-top-left-radius: $search-filter-radius;
+  border-top-right-radius: $search-filter-radius;
   border: 1px solid  $search-filter-border-color;
   overflow: hidden;
 }
 
+.svg-icon-tooltip {
+  width: $further-info-icon-width;
+  height: $further-info-icon-height;
+  padding: 2px 1px 0 1px;
+  fill: $further-info-background-color;
+}
+
 .search-filter--container {
   @include row(12);
   padding-top: $baseline-unit*3;
 }
 
+.search-filter--container .l-results__filter {
+  width: 100%;
+}
+
 .search-filter__form {
   // override Dough class 'form'
```

Comment:
There's already a dough modifier for this: https://github.com/moneyadviceservice/frontend/blob/master/app/assets/stylesheets/components/dough_theme/form/_form.scss#L21-L23